packages/devtools_app/lib/src/eval_on_dart_library.dart

```diff
@@ -56,14 +63,16 @@ class EvalOnDartLibrary {
 
   final Set<String> _candidateLibraryNames;
   final VmServiceWrapper service;
-  Completer<LibraryRef> _libraryRef;
   Future<void> _initializeComplete;
   StreamSubscription selectedIsolateStreamSubscription;
 
   String get isolateId => _isolateId;
   String _isolateId;
 
+  Completer<LibraryRef> _libraryRef;
   Future<LibraryRef> get libraryRef => _libraryRef.future;
+  Completer<LibraryRef> get libraryRefCompleter => _libraryRef;
```

Comment:
do you really need the completer rather than just the Future? Completer should be an implementation detail.